index.html

```diff
@@ -263,6 +263,28 @@ <h2 class="kn_1">
       </li>
     </ul>
   </section>
-</body>
 
+  <section class="contact-form">
```

Comment:
- [ ] As requested by the previous reviewer, The contact form should have the same background images as on the Figma website. Please correct the little bit top margin of the `contact me` text also.
- Your website 
![Screenshot from 2022-03-30 17-25-09](https://user-images.githubusercontent.com/26767383/160830030-325941e1-e5de-4e3b-843d-32488dc7381a.png)




- Figma website
![Screenshot from 2022-03-30 17-26-46](https://user-images.githubusercontent.com/26767383/160829238-5f5a5626-a0ff-4570-8ef3-ae32d8050ba2.png)